src/stats.c

```diff
@@ -328,6 +398,43 @@ stats_server_t *stats_server_create(struct ev_loop *loop,
 					goto server_create_err;
 			}
 		}
+
+		if (config->send_self_stats) {
+			/**
+			  * Only single config for monitor section
+			  */
+			struct additional_config *stat = config->sstats->data[0];
```

Comment:
Probably want to enforce only one sstats entry in the config? (I think we do this already but could lead to later surprises)